components/brave_wallet_ui/page/screens/composer_ui/from_asset/from_asset.style.ts

```diff
@@ -51,3 +53,15 @@ export const SelectTokenAndInputRow = styled(Row)`
 export const NetworkAndFiatRow = styled(Row)`
   min-height: 22px;
 `
+
+export const InfoIcon = styled(Icon).attrs({
+  name: 'info-outline'
+})`
+  --leo-icon-size: 16px;
+  color: ${leo.color.icon.interactive};
+  margin-right: 8px;
```

Comment:
can we set a `gap` on the row that contains this instead?

Reply:
Am not able to use `gap` in the case since the `row` eventually will wrap for the `Panel` and will cause unwanted `vertical` gaps.